### PR TITLE
Release 0.49.0

### PR DESCRIPTION
**Pull request by lfittl**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+## 0.49.0      2023-02-27
+
+* Update pg_query_go to v4 / Postgres 15 parser
+  - Besides supporting newer syntax like the MERGE statement, this parser
+    update also drops support for "?" replacement characters found in
+    pg_stat_statements output before Postgres 10
+  - Postgres 10 is now the minimum required version for running the collector
```
Comment (keiko713) [inline CHANGELOG.md:9]: Is this... worth a separate line?

**Comment by keiko713** [comment]
I think we have more places to update the version number, like `packages/Makefile`: https://github.com/pganalyze/collector/pull/366/files

File: CHANGELOG.md (hunk 1 above)
Comment (lfittl) [reply CHANGELOG.md]: Yep, good idea

**Review by keiko713 — APPROVED** [review]
🚢

**Review by msakrejda — APPROVED** [review]

**Pull request merged**